src/date/expand-pattern.js

```diff
@@ -38,6 +39,89 @@ return function( options, cldr ) {
 		);
 	}
 
+	function getBestMatchPattern( path, skeleton ) {
+		var availableFormats, ratedFormats, format;
```

Comment:
I assume you want to define `pattern` as a variable local to this function, so it's not mixed up with `pattern` defined by the parent function.